src/lib/serviceRoutines/getAttributesForEntityType.cpp

```diff
@@ -49,6 +49,7 @@
 *
 * URI parameters:
 *   - attributesFormat=object
+*   - options=noAttrDetail
```

Comment:
This was just added as the signature for mongoAttributesForEntityType was changed (adding the 'noAttrDetail' parameter).
If what you say is true, that the code for the attr-type-list is not even executed for v1 (and I don't doubt your word :-)), then this should change, removing the option as a possible URO param and false sent to mongoAttributesForEntityType